theme/theme.ts

```diff
@@ -0,0 +1,128 @@
+
+const themeDefault = {
```

Comment:
Então dentro desse arquivo, você pode criar um `export type Theme = { ... }` com os valores definidos em `ComponentsProvider.tsx`.

E então, você pode tipar a `const` `themeDefault` fazendo:

```suggestion
const themeDefault: Theme = { ... }
```

E você fazendo o export do type Theme você pode utilizar ele, no `ComponentsProvider.tsx` para tipar a prop `theme`.


me chama se tiver com duvida, que fazemos juntas ;)